lib/routes/bulk-get.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+var utils = require('../utils');
+
+module.exports = function (app) {
+  utils.requires(app, 'routes/db');
+
+  // Bulk docs operations
+  app.post('/:db/_bulk_get', utils.jsonParser, function (req, res, next) {
+
+    var opts = req.body || {};
+
+    if (Array.isArray(req.body)) {
+      return utils.sendJSON(res, 400, {
+        error: "bad_request",
+        reason: "Request body must be a JSON object"
+      });
+    }
+    if (!req.body.docs) {
+      return utils.sendJSON(res, 400, {
+        error: "bad_request",
+        reason: "Missing JSON list of 'docs'"
+      });
+    }
+
+    if (typeof req.db.bulkGet !== 'function') {
+      return utils.sendJSON(res, 400, {
+        error: "bad_request",
+        reason: "bulkGet not supported by this version of PouchDB"
+      });
+    }
+
+    // these can be specified via query, e.g. ?revs=true
```

Comment:
@marten-de-vries because only these two are "officially" [allowed](http://developer.couchbase.com/mobile/develop/references/sync-gateway/rest-api/database/post-bulk-get/index.html)  to use there while others brought you to the terra incognito. For instance, CouchDB implementation unintentionally happens to allow all the most of the others document params like meta, r, etc, but reject requests with open_revs and atts_since params in order to prevent confusion.